src/scenes/home/header/sideNav/sideNav.js

```diff
@@ -80,7 +79,6 @@ class SideNav extends Component {
               isExternal
```

Comment:
You didn't put it here, but can you remove this line?

/code_schools isn't external, obviously.